charts/ndb-operator/templates/serviceaccount-ndb-operator-controller-manager.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: {{ include "ndb-operator.serviceAccountName" . }}
+  name: ndb-operator-controller-manager
```

Comment:
Service account should not be called the "controller-manager", right? 
Shouldn't it be linked to the name that we let the user provide in the values.yaml?

Reply:
Updated the name to "ndb-operator-service-account" manually. This was auto-generated from the "make deploy" output, meaning it needs to be fixed there so we will not need to update this in every release manually.